pretyping/typing.ml

```diff
@@ -514,13 +514,15 @@ let sort_of env sigma c =
 
 (* Try to solve the existential variables by typing *)
 
-let type_of ?(refresh=false) env sigma c =
+let type_of refresh env sigma c =
```

Comment:
Same remark about labels.

Reply:
I concur, why leaving aliases in the first place